app/components/new-sign-up/form.hbs

```diff
@@ -3,18 +3,16 @@
     {{this.label}}
   </h2>
   <div class="user-details__field">
-    <Input @value={{get @userDetails @state}} type="text" placeholder="Darth" {{on 'input' this.inputFieldChanged}} data-test-signup-form-input/>
+    <Input @value={{get @userDetails @currentStep}} type="text" placeholder="Darth" {{on 'input' this.inputFieldChanged}} data-test-signup-form-input/>
```

Comment:
The placeholder here seems static for any kind of input. Will we be showing the same placeholder for any step?

Reply:
Updated